docs/AUTHENTICATION.md

```diff
@@ -35,48 +35,45 @@ These are some of the terms that you will encounter in this document and other r
+## CDS Supported Authenticated Flows
 
-Currently Cds supports a number of authentication flows. Based on your requirements choose the one that best fits your needs. The following subsections are more technical and implementation oriented than the first part of the document.
-
-### Authorization Code Flow with PKCE
-
-If you are developing any Javascript/Browser (SPA) based applications or native mobile applications, with the user (resource owner) trying to access resources, use this flow. This flow adds an extra layer of security on top of the OIDC implicit flow by:
-
-1. Requiring a client-verified code exchange for access token
-2. By not returning the access token in a redirect URI.
-
-In this OIDC flow, no refresh token is provided.
-
-Authorization Code Flow supports silent refresh, which makes it possible to receive a new access token while the user is both using the application and logged in with the Identity Provider in the same browser session. This is done behind the scenes without interrupting the user experience.
-
-The sample for this authentication flow can be found [here for DotNet](https://github.com/osisoft/sample-adh-authentication_authorization-dotnet), [here for NodeJS](https://github.com/osisoft/sample-adh-authentication_authorization-nodejs), and [here for Python](https://github.com/osisoft/sample-adh-authentication_authorization-python).
+Currently CDS supports two authentication flows. Based on your requirements choose the one that best fits your needs. The following subsections are more technical and implementation oriented than the first part of the document.
 
 ### Client Credential Flow
 
-If you are writing software (client) to communicate with Cds without the presence of a user, then this is authentication flow you should follow. This flow was created for machine to machine communication.
+If you are writing software (client) to communicate with CDS without the presence of a user (resource owner), then this is the authentication flow you should follow. This flow was created for machine to machine communication.
 
-A client is any software that a resource owner uses to access his resources on a remote server. In this case the client itself is the resource owner, and there is no actual user to perform the authentication process. The client uses his Client Id and Client Secret to authenticate against Cds and is awarded an Access Token. It is assumed that the client stores the Client Secret in a safe location, and uses cryptographically secure channels -read https- to communicate with Cds. Cds only supports communication over https. No Refresh Token is awarded.
+The client uses its Client Id and Client Secret to authenticate against CDS and is awarded an Access Token. It is assumed that the client stores the Client Secret in a safe location, and uses cryptographically secure channels -read https- to communicate with CDS. CDS only supports communication over https. No Refresh Token is awarded.
```

Comment:
Another nitpick, is there a reason `Client Id`, `Client Secret`, `Access Token` and `Refresh Token` are capitalized here?  These should probably be lower-cased